app/controllers/transactions_controller.rb

```diff
@@ -11,6 +11,33 @@ def initialize
     super
   end
 
+  def index
+    account_ids = current_user.account_users.map(&:account_id)
+    order_details = OrderDetail.where(account_id: account_ids).joins(:order)
```

Comment:
Just curious, can we use `User#administered_order_details`? That's what the in dispute uses. Not sure if that means the same thing or not?

Reply:
That means a more specific thing--based on the way we're displaying account transactions currently, users should be able to see transactions on any account they have access to. `User#administered_order_details`, by contrast, would show transactions for only the accounts where the user has an admin user role (i.e. not just "purchaser").

I made the call towards preserving current behavior, but it is inconsistent with how things work with `in_dispute`--if this is a concern we could run it past the client and see what they think.

Reply:
If it maintains current behavior, then let's stick with that. Thanks for double checking; I wasn't sure the difference.